uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -202,6 +208,37 @@ def test_can_enable_false_on_unsupported_kernel_flavor(
         )
         assert (msg, "") == capsys.readouterr()
 
+    @pytest.mark.parametrize(
```

Comment:
@OddBloke I added this test to validate above and below cut line for minKernelVersion comparisons.

Reply:
Thanks!